src/main/resources/pdiresources/jobs/job-for-encounter.kjb

```diff
@@ -0,0 +1,379 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we really need a job that only starts and runs the transformation? why not as other transformations, we probably only need the transformation.

Reply:
the philosophy of  our  migrator tool  is to have  job(s) orchestrating  the transformations... as  tool is programmed  we can't reach transformations without job

Reply:
which i think is why we wrote the entry (control-center) job, otherwise we will end up having a job for every transformation which i don't think is what we want, we want probably one job orchestrating all validation transformations and another job doing the same for merge transformations

Reply:
good observation! so it means each time we add a transformation to the tool we have to  edit `control-center.kjb` to let him  know that there is one  more transformation to be executed?

Reply:
not really, rather the jobs it invokes, either validation or transformation ones.

Reply:
So in this  case I think we can put jobs in the  **control-center** to  maintain  the abstraction. what do  you  think?